src/components/Slider/Slider.js

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from "react";
+import "./slider.css";
+import { useFeaturedBanners } from "../../utils/hooks/useFeaturedBanners";
+import "react-image-gallery/styles/css/image-gallery.css";
+import ImageGallery from "react-image-gallery";
+
+const Slider = () => {
+  const { data, isLoading } = useFeaturedBanners();
+  const [items, setItems] = useState([]);
+
+  useEffect(() => {
+    if (data.results !== undefined) {
+      let items = data.results.map((row) => {
```

Comment:
`let` is used when the value of the variable is going to change, `const` is used when the value won't change, including the values in arrays and objects

```suggestion
      const items = data.results.map((row) => {
```